### PR TITLE
#253 changed size of about container on HackAUBG website

### PR DESCRIPTION
**Pull request by Pessy1**
I worked on changing the size of the about container to 80% of the page instead of 300vh.

### CODE REVIEW

File: packages/web/src/components/spa/Navigation/DesktopNav/desktop_navbar.css [hunk 1]
```diff
@@ -68,6 +68,16 @@
     align-items: center;
 }
 
+@media only screen and (max-width: 1073px) {
```
Comment (SuelaCark) [inline packages/web/src/components/spa/Navigation/DesktopNav/desktop_navbar.css:71]: All changes from the file desktop_navbar.css should be deleted from here. You have probably created this branch from another branch. Please next time checkout to master before creating a new branch.

**Review by SuelaCark — CHANGES_REQUESTED** [review]
Please just make the requested changes. Everything else looks good. 
Good job Slavcho.

**Comment by SuelaCark** [comment]
@Aleks1908 Please review this PR^^

**Review by Aleks1908 — APPROVED** [review]
LGTM

**Comment by SuelaCark** [comment]
@asynchroza Please review and merge this branch to master

**Pull request merged**